donkeycar/parts/tub_v2.py

```diff
@@ -109,24 +117,79 @@ class TubWriter(object):
     """
     A Donkey part, which can write records to the datastore.
     """
+
+    class TubRecordCached:
```

Comment:
We don't need this. Make a utility function if you absolutely need it.

Reply:
Can you advice how to make a utility function for its job?